Engine/Colliders/Collider.cs

```diff
@@ -0,0 +1,54 @@
+﻿using AntEngine.Maths;
+
+namespace AntEngine.Colliders
+{
+    /// <summary>
+    /// Abstract collider allows for the calculation of collision between primitive geometrical shapes.
+    /// </summary>
+    public abstract class Collider
+    {
+        /// <summary>
+        /// Position, rotation and scale of the collider.
+        /// </summary>
+        private Transform ColliderTransform;
+        private Transform ParentTransform;
+
+        /// <summary>
+        /// Constructor for a default collider without specifying any coordinates.
+        /// Will initialize a collider at 0,0 with a scale of 1,1 with its parent possessing the same coordinates.
+        /// </summary>
+        public Collider() : this(new Transform(), new Transform()) { }
```

Comment:
A default constructor shouldn't exist I think, a collider is linked to a transform that already exists